sys/include/net/gnrc/rpl.h

```diff
@@ -9,8 +9,9 @@
  */
 
 /**
- * @defgroup    net_gnrc_rpl  New RPL
- * @ingroup     net
+ * @defgroup    net_gnrc_rpl  RPL
+ * @ingroup     net_gnrc
+ * @brief       RPL implementation for GNRC
```

Comment:
I go them through backwards, I hope you don't mind, but again: These are not helper functions. They are part of the implementation and provide a huge chunk of the functionality of RPL. Helper functions to me are auxiliary functions that can easily replaced by an external max. 5-line code with a deeper understanding of the code/protocol or that unify something otherwise very tedious (like printing the type).